task/07-yield-tasks.js

```diff
@@ -47,7 +54,13 @@ function* get99BottlesOfBeer() {
  *
  */
 function* getFibonacciSequence() {
-    throw new Error('Not implemented');
+    let a = 0, b = 1;
+    while(b < Number.MAX_VALUE) {
```

Comment:
it is necessary to observe indents between operators and brackets.
https://javascript.info/coding-style